yubikey-luks-open

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+DISK="/dev/sda3"
+NAME="yubikey-luks"
+DBG=0
+TMP_FILE=/tmp/new_key
+set -e
+. /etc/ykluks.cfg
+
+if [ "$(id -u)" -ne 0 ]; then
+  echo "You must be root." 1>&2
+  exit 1
+fi
+
+while getopts ":d:n:hv" opt; do
+  case $opt in
+	d)
+		DISK=$OPTARG
+		echo "setting disk to $OPTARG."
+		;;
+	n)
+		NAME=$OPTARG
+		echo "setting name to $OPTARG."
+		;;
+		v)      DBG=1
+		echo "debugging enabled"
+		;;
+	h)
+		echo 
+		echo " -d <partition>: set the partition"
+		echo " -n <name>     : set the container name"
```

Comment:
To me the test "set the partition" and "set the container name" are misleading.
Because the disk partition exists and the containers are already named.
The user will *choose* an existing container and partition.

I would call this

"choose the partition to unlock" and "choose the name of the container to unlock" if my understanding is right.

Reply:
No, before unlocking LUKS encrypted partition only appears as /dev/sdaX . Only while opening/unlocking you assign the LUKS name to this partition and only after then it appears as /dev/mapper/<chosen-name> in decrypted form

So there is no existing container name before you open it.

Reply:
Point taken!

But then it would be good to differentiate between the *new* name and the *existing* partition.

Reply:
Ok. Can you edit and adjust the wording yourself as you like?

Reply:
I edited it myself. Hope it's clearer now.